packages/elements/list/src/withList.spec.tsx

```diff
@@ -326,11 +326,7 @@ describe('withList', () => {
               </hlic>
             </hli>
           </hul>
-          <hul>
-            <hli>
-              <hlic>level 2</hlic>
-            </hli>
-          </hul>
+          <hp>level 2</hp>
```

Comment:
This was to make sure we don't catch any list item outside the current list, paragraph tests should be covered by another suite.

```suggestion
          <hul>
            <hli>
              <hlic>level 2</hlic>
            </hli>
          </hul>
```

Reply:
Whit my changes if 2 list follows each other and you hit the delete at the end of the first list it merges the two list together, so this test fails, because the result would be
```
<editor>
    <hul>
        <hli>
            <hlic>
                level 1<cursor />
            </hlic>
        </hli>
        <hli>
            <hlic>level 2</hlic>
        </hli>
    </hul>
</editor>
```
(but there is a test for this in [getListDeleteForward.spec.tsx](https://github.com/udecode/plate/blob/fc9ad9a9d5d46ac3359baa1b6b9226969cfdb7ed/packages/elements/list/src/getListDeleteForward.spec.tsx#L173))

Reply:
Took another look - what we did there was intended behavior at the time. Maybe the change makes sense, but the test in this context is ok. @dylans you good with the new behavior?